lib/Activity.py

```diff
@@ -557,7 +557,7 @@ def run_stages(self, config):
 
         rollout_percentage = config.args.get("concurrent_management_rollout_percentage")
         if rollout_percentage:
-           payload["input_parameters"]["concurrent_management_rollout_percentage"] = rollout_percentage
+           payload["input_parameters"]["concurrent_management_rollout_percentage"] = int(rollout_percentage)
```

Comment:
you might want this in a try block unless we can add input validation because they could pass you in a percentage of "foo".

Reply:
![image](https://user-images.githubusercontent.com/123126294/215883198-c4ddb609-e5ea-4976-a2ca-60d3d8ce1fc2.png)
```
./iuf -i input.yaml run -r pre-install-check -cmrp foo
usage: iuf run [-h] [-b BEGIN_STAGE] [-e END_STAGE] [-r RUN_STAGES [RUN_STAGES ...]] [-s SKIP_STAGES [SKIP_STAGES ...]]
               [-bc BOOTPREP_CONFIG_MANAGED] [-bm BOOTPREP_CONFIG_MANAGEMENT] [-bpcd BOOTPREP_CONFIG_DIR] [-rv RECIPE_VARS]
               [-sv SITE_VARS] [-mrs {reboot,stage}] [-cmrp CONCURRENT_MANAGEMENT_ROLLOUT_PERCENTAGE]
               [--limit-managed-rollout LIMIT_MANAGED_ROLLOUT [LIMIT_MANAGED_ROLLOUT ...]]
               [--limit-management-rollout LIMIT_MANAGEMENT_ROLLOUT [LIMIT_MANAGEMENT_ROLLOUT ...]]
               [-mrp MASK_RECIPE_PRODS [MASK_RECIPE_PRODS ...]]
iuf run: error: argument -cmrp/--concurrent-management-rollout-percentage: invalid int value: 'foo'
```

Reply:
I'll commit that change as a review suggestion in a few minutes.